core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -658,6 +658,7 @@ public enum BuiltInMethod {
   ARRAYS_OVERLAP(SqlFunctions.class, "arraysOverlap", List.class, List.class),
   ARRAYS_ZIP(SqlFunctions.class, "arraysZip", List.class, List.class),
   SORT_ARRAY(SqlFunctions.class, "sortArray", List.class, boolean.class),
+  ARRAY_INSERT(SqlFunctions.class, "arrayInsert", List.class, Integer.class, Object.class),
```

Comment:
Put this line to `https://github.com/apache/calcite/pull/3320/files#diff-5207c1dd4af072b3ecbebf9bf87ab0677171bd47309eda809013ef66ae9f77d6R655`. Alphabetic order.

Reply:
fixed